notebooks/official/prediction/pytorch_image_classification_with_prebuilt_serving_containers.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>Question from me, not necessarily relevant to the user. The overview said "minimal configuration" so I would have expected the default image_handler, if it was possible to use. </p><p> </p><p>Did we specifically want to demontrate Custom Handler because it is a common use case? Or is there a particular reason that one is being used here (perhaps to limit the top3)?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1592/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>It's possible to use default image_handler for this case. Because it's typically for users to have some customization for model handlers, the sample coder here demonstrate the way to add custom code to a default hander so users can do customization with minimum configuration.</p>